### PR TITLE
Add tests

### PR DESCRIPTION
**Pull request by nbirillo**
- Add simple tests to check the correctness of the brackets. The parser adds brackets even if they were not in the source code. For example: a = 5 / 10 -> a = (5 / 10)
- Add a logger, different utils, configure the logger for the tests
- Tests failed because I found a bug in the parser (Nikita should check it)

### CODE REVIEW

File: src/main/util/const.py [hunk 1]
```diff
@@ -1,9 +1,22 @@
...
+# Copyright (c) Aniskov N., Birillo A.
 
 import os
-from pathlib import Path
+from enum import Enum
 
-ROOT_DIR = Path(__file__).resolve().parents[3]
-TEST_RESOURCES_PATH = os.path.join(ROOT_DIR, 'src/test/resources')
+# src folder
+ROOT_DIR = os.path.join(os.path.dirname(os.path.abspath(__file__)), '../../')
```
Comment (aniskoff) [inline src/main/util/const.py]: Why are you think this is better than `ROOT_DIR = Path(__file__).resolve().parents[3]` ? The previous one is easier to read than the one that is now, don't you think so?

**Review by aniskoff — COMMENTED** [review]
Not quite sure why you decided to convert `test/resources` directory to Python Package i.e. you added `__init__.py` files inside `test/resources`. As far as I can see you are not using `test/resources` as package (i.e. you don't importing anything from it). If so, i think it would be reasonable to convert `test/resources` back to simple directory (i.e. remove all `__init__.py` files from it). What do you think?

File: src/test/python/inverse_parser/test_brackets.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+        actual_out = self.__calculate_expression_safely(self.__get_python_popen_args(source_file))
+        remove_file(source_file)
+        return actual_out
+
+    def __create_source_file(self, source_code: str) -> str:
+        source_code_file = os.path.join(self.EXPRESSIONS_SOURCES_PATH, 'brackets_test.py')
+        create_file(source_code, source_code_file)
+        return source_code_file
+
+    @pytest.mark.parametrize('source', [_ for _ in EXPRESSIONS_SOURCES])
```
Comment (aniskoff) [inline src/test/python/inverse_parser/test_brackets.py]: `[_ for _ in EXPRESSIONS_SOURCES]` is equivalent to `EXPRESSIONS_SOURCES` isn't it? So why not to use just `EXPRESSIONS_SOURCES`?

File: src/main/util/const.py (hunk 1 above)
Comment (nbirillo) [reply src/main/util/const.py]: Sorry, I did it automatically, thanks

File: src/test/python/inverse_parser/test_brackets.py (hunk 2 above)
Comment (nbirillo) [reply src/test/python/inverse_parser/test_brackets.py]: You are right, of course, thanks!

**Comment by aniskoff** [comment]
Great job, thanks!

**Pull request merged**